tests/web/Widget.cc

```diff
@@ -0,0 +1,146 @@
+//  This file is part of Empirical, https://github.com/devosoft/Empirical
```

Comment:
Love the new file naming scheme on `test_widget` -> `Widget.cc`... yay for uniformity